frontend/src/email.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
I think you meant to have this file on your other [branch](https://github.com/CryptoCardsETH/cryptocards/pull/80)?